arrows/qt/EmbeddedPipelineWorker.cxx

```diff
@@ -49,6 +51,34 @@ class EmbeddedPipeline : public kwiver::embedded_pipeline
   bool outputConnected_ = false;
 };
 
+// ----------------------------------------------------------------------------
+EmbeddedPipeline
+::EmbeddedPipeline( RequiredEndcaps endcaps )
+  : endcaps_{ endcaps }
+{
+  auto ss = []( QString const& qs ){
```

Comment:
This looks like an obscure way to convert a `QString` to a `std::string`.  What is the purpose of using this compared to one of the built-in Qt functions for this?

Reply:
It's the same as [`stdString`](https://github.com/Kitware/qtextensions/blob/0ff5b486f08435c8cdfbc9f05d2f104f63b16aed/core/qtStlUtil.h#L37-L41). `QString::toStdString` assumes that the system locale is UTF-8. That's probably safe on *nix, but has historically been incorrect on Windows; do you know if that has changed?

Reply:
I see.  It's in qtExtensions and we are not depending on qtExtensions here.  I don't know about the UTF-8 issues.

Reply:
>  It's in qtExtensions and we are not depending on qtExtensions here.

Right (otherwise I would have just used `stdString`).

Reply:
I think it could use a comment to say what `ss` does, or maybe just rename it to `stdString`.  It was not immediately obvious.  If you are adding a comment, a quick note about why not `QString::toStdString` might also be beneficial.  After I figured what it was doing, my next questions was "why are we not using standard Qt functions for this?".

Reply:
Done.